src/test/java/com/afornalik/ox/board/TestBoard.java

```diff
@@ -2,30 +2,33 @@
 
 import org.testng.annotations.Test;
 
-import java.util.TreeMap;
-
+import static org.assertj.core.api.Assertions.*;
 import static org.testng.Assert.*;
 
 @Test
 public class TestBoard {
 
     private static final int BOUND_FOUR = 4;
     private static final int BOUND_THREE = 3;
+    private static final int INDEX_1 = 1;
+    private static final int MINUS_INDEX = -1;
+    private static final FieldStatus FIELD_STATUS_X = FieldStatus.X;
+    private static final FieldStatus FIELD_STATUS_O = FieldStatus.O;
```

Comment:
Just wondering - why constants?

Reply:
More descriptive but, hm.. enum is enough descriptive itself. actually i stay with this version.

Reply:
Ok but, if they are used only in this test class should they be static? And if not, they are not constants anymore. Just a suggestion.

Reply:
I think i saw this way of constant declaration in tests. I must check this, anyway actually i stay with this version.